.claude/scripts/ai-review/prioritizer.py

```
@@ -0,0 +1,299 @@
+"""
+AI Review Prioritizer
+
+투표 시스템 + Zero-Tolerance 규칙 기반 우선순위 계산
+- 3봇 합의 → Critical
+- 2봇 합의 → Important
+- 1봇만 → Suggestion
+- Zero-Tolerance 위반 → 자동 Critical
+"""
+
+import re
+from typing import List, Dict
```

Ruff (0.14.3)

`typing.List` is deprecated, use `list` instead

(UP035)